src/main/java/uk/gov/hmcts/reform/cwrdapi/domain/CaseWorkerProfile.java

```diff
@@ -91,4 +89,38 @@
     @JoinColumn(name = "user_type_id", referencedColumnName = "user_type_id",
             insertable = false, updatable = false, nullable = false)
     private UserType userType;
+
+    public CaseWorkerProfile(String caseWorkerId, String firstName, String lastName, String emailId, Long userTypeId,
```

Comment:
Can't we use the builder pattern to create objects instead of adding multiple constructor?

Reply:
I tried this first and got errors during build. Something to do with the fields relating to database columns I think.

I will reproduce it and let you know exactly what the error is..

Reply:
As discussed, the builder pattern is giving us issues. Hence a constructor is the only way.
Thanks

Reply:
Sorry, why we need two constructors? is not @allargsconstructor sufficient ? Correct me if I am wrong here

Reply:
I believe you are correct, I think I was using the constructor until I realised I required all fields. Then overlooked just using the @AllArgsConstructor annotation instead. Will change and commit.

Reply:
cheers